Tests/Command/LoadDataFixturesDoctrineCommandTest.php

```diff
@@ -29,11 +31,13 @@ public function testInstantiatingWithoutManagerRegistry() : void
             if (PHP_VERSION_ID >= 80000) {
                 $this->expectExceptionMessage(
                     <<<'MESSAGE'
-Doctrine\Bundle\DoctrineBundle\Command\DoctrineCommand::__construct(): Argument #1 ($doctrine) must be of type Doctrine\Persistence\ManagerRegistry, null given, called in /home/runner/work/DoctrineFixturesBundle/DoctrineFixturesBundle/Command/LoadDataFixturesDoctrineCommand.php on line 49
+Doctrine\Bundle\DoctrineBundle\Command\DoctrineCommand::__construct(): Argument #1 ($doctrine) must be of type Doctrine\Persistence\ManagerRegistry, null given, called in /home/runner/work/DoctrineFixturesBundle/DoctrineFixturesBundle/Command/LoadDataFixturesDoctrineCommand.php on line 51
```

Comment:
Is the line number relevant for the error message check? It seems like an unstable test when changes are happening.

Reply:
It is unstable indeed, I remember having to change it in the past. Let me try refactoring it.